apps/android/app/src/main/java/chat/simplex/app/views/newchat/ContactConnectionInfoView.kt

```diff
@@ -43,21 +45,24 @@ fun ContactConnectionInfoView(
       }
     }
   }
+  val context = LocalContext.current
   ContactConnectionInfoLayout(
     connReq = connReqInvitation,
     contactConnection,
+    connIncognito = chatModel.incognito.value,
```

Comment:
on pending connection view (opened from chat list) incorrect shared profile text is shown - it depends on incognito mode instead of connection state